scripts/mixins/fomor_hate.lua

```diff
@@ -0,0 +1,17 @@
+require("scripts/globals/mixins");
+
+g_mixins = g_mixins or {};
+
+g_mixins.fomor_hate = function(mob)
+    mob:addListener("DEATH", "FOMOR_HATE_DEATH", function(mob, player, isKiller)
+        local hate = player:getVar("FOMOR_HATE");
+        local adj = mob:getLocalVar("fomorHateAdj");
+        if (adj == 0) then adj = 2 end; -- most fomor add 2 hate
+        hate = hate + adj;
+        hate = math.max(hate, 0);
+        hate = math.min(hate, 60);
```

Comment:
https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/utils.lua#L19 might be easier to follow

Reply:
we had a 2 people vote on discord and decided utils.clamp is easier to follow